lib/mvi/responses/icn_with_aaid_parser.rb

```diff
@@ -28,17 +27,13 @@ def initialize(extension)
       #
       def without_id_status
         return if extension.nil?
-        return if invalid_id_status?
+        return unless id_status == 'P'
```

Comment:
Should I leave this line?
This class is only used in `MVI::Responses::IdParser`, which now only passes in IDs with a status of 'P'